numba/help/inspector.py

```diff
@@ -0,0 +1,429 @@
+
+            else:
+                self.print("   - defined by ``{}``".format(str(tcls)))
+        self.print()
+
+        if alias:
+            self.print("   alias to: ``{}``".format(alias))
+        self.print()
+
+    def write_unsupported_item(self, modname, itemname):
+        pass
+
+    def write_statistic(self, stat):
+        if stat.supported == 0:
+            self.print("This module is not supported.")
+        else:
+            msg = "Not showing {} unsupported functions."
+            self.print(msg.format(stat.unsupported))
+            self.print()
+            self.print(stat.describe())
+        self.print()
+
+
+def _format_module_infos(formatter, package_name, mod_sequence, target=None):
+    """Format modules.
+    """
+    formatter.title('Listings for {}'.format(package_name))
+    alias_map = {}  # remember object seen to track alias
+    for mod in mod_sequence:
+        stat = _Stat()
+        modname = mod.__name__
+        formatter.begin_module_section(formatter.escape(modname))
+        for info in inspect_module(mod, target=target, alias=alias_map):
+            nbtype = info['numba_type']
+            if nbtype is not None:
+                stat.supported += 1
+                formatter.write_supported_item(
+                    modname=formatter.escape(info['module'].__name__),
+                    itemname=formatter.escape(info['name']),
+                    typename=formatter.escape(str(nbtype)),
+                    explained=formatter.escape(info['explained']),
+                    sources=info['source_infos'],
+                    alias=info.get('alias'),
+                )
+
+            else:
+                stat.unsupported += 1
+                formatter.write_unsupported_item(
+                    modname=formatter.escape(info['module'].__name__),
+                    itemname=formatter.escape(info['name']),
+                )
+
+        formatter.write_statistic(stat)
+        formatter.end_module_section()
+
+
+def write_listings(package_name, filename, output_format):
+    """Write listing information into a file.
+
+    Parameters
+    ----------
+    package_name : str
+        Name of the package to inspect.
+    filename : str
+        Output filename. Always overwrite.
+    output_format : str
+        Support formats are "html" and "rst".
+    """
+    package = __import__(package_name)
+    if hasattr(package, '__path__'):
+        mods = list_modules_in_package(package)
+    else:
+        mods = [package]
+
+    if output_format == 'html':
+        with open(filename + '.html', 'w') as fout:
+            fmtr = HTMLFormatter(fileobj=fout)
+            _format_module_infos(fmtr, package_name, mods)
+    elif output_format == 'rst':
+        with open(filename + '.rst', 'w') as fout:
+            fmtr = ReSTFormatter(fileobj=fout)
+            _format_module_infos(fmtr, package_name, mods)
+    else:
+        raise ValueError("{} is not supported".format(output_format))
```

Comment:
```suggestion
        raise ValueError("Output format '{}' is not supported".format(output_format))
```
(with linewrap as the above is over 80chars)